.devcontainer/on_create.sh

```diff
@@ -2,7 +2,5 @@
 sudo apt update
 sudo apt install -y netcat
 sudo add-apt-repository -y ppa:deadsnakes/ppa
-sudo apt install -y python3.11
-curl -sSL https://install.python-poetry.org | python3.11 -
-# chromadb requires SQLite > 3.35 but SQLite in Python3.11.9 comes with 3.31.1
-sudo cp /opt/conda/lib/libsqlite3.so.0 /lib/x86_64-linux-gnu/libsqlite3.so.0
+sudo apt install -y python3.12
+curl -sSL https://install.python-poetry.org | python3.12 -
```

Comment:
Python 3.12 includes SQLite 3.40

Reply:
Interesting. That's a good thing! I hope this works.

FYI, if this PR does work, we probably need to review and/or revert this: https://github.com/All-Hands-AI/OpenHands/pull/3713